### ISSUE TITLE
Text (AWT font) is not stretched with TextAttribute.TRACKING set greater than zero 

### ISSUE THREAD

**Issue by flaxx** [LibrePDF/OpenPDF#796]
**Describe the bug**
Text (AWT font) in the pdf document is not stretched with TextAttribute.TRACKING  set greater than zero.

**To Reproduce**
1. Run attached Java code (Main class App.java) with JDK >= 10

**Expected behavior**
When TextAttribute.TRACKING is set greater then zero, the text in the pdf document should be streched.

**Screenshots**
see attached Screenshots.pdf

**System**
 - OS: Windows 10, Redhat Linux Enterprise 8
 - Used Font: Font("Arial", Font.PLAIN, 15)
 - Bug starts with JDK >= 10; 
 - Redhat openJDK and Oracle openJDK

**Additional context**
Screenshots for JDK8 and JDK11
[Screenshots.pdf](https://github.com/LibrePDF/OpenPDF/files/9382175/Screenshots.pdf)

Sourcecode to create the pdf
[java.zip](https://github.com/LibrePDF/OpenPDF/files/9382182/java.zip)

Maven Dependency
```
<dependency>
            <groupId>com.github.librepdf</groupId>
            <artifactId>openpdf</artifactId>
            <version>1.3.29</version>
</dependency>
```

**Comment by flaxx** [comment]
This bug ist related to 
"Font.getStringBounds calculates wrong width for TextAttribute.TRACKING other than 0.0"
https://bugs.openjdk.org/browse/JDK-8304350

**Comment by flaxx** [comment]
Bug was caused by the openJDK, see https://bugs.openjdk.org/browse/JDK-8304350
JDK-Bug is fixed in 11.0.20 , 17.0.8, 20.0.2
